tjreports/administrator/controllers/tjreport.php

```diff
@@ -52,49 +52,25 @@ public function getparams()
 		$result = $model->getparams();
 	}
 
-	/**
-	 * Gets the URL arguments to append to an item redirect.
-	 *
-	 * @param   integer  $recordId  The primary key id for the item.
-	 * @param   string   $urlVar    The name of the URL variable for the id.
-	 *
-	 * @return  string  The arguments to append to the redirect URL.
-	 *
-	 * @since   1.6
-	 */
-	protected function getRedirectToItemAppend($recordId = null, $urlVar = 'id')
```

Comment:
Yes , Because I had send the MR for that code and now I got solution on that in minimum lines so replaced previous code with new one.